lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -418,6 +419,78 @@ def inspect
         end
 
         private
+        
+        # Integrates the targets of the user projects with the on demand resources
+        #
+        # we create groups like:
+        #
+        # on_demand_resources
+        # ----- tag1
+        # ------------ user resource dir
+        # ----- tag2
+        # ------------ user resource dir
+        #
+        # we only add user resource dir in resources build phase.
+        #
+        # @return [void]
+        def add_on_demand_resources
+
+          attributes = target.user_project.root_object.attributes
+          asset_tags = (attributes["KnownAssetTags"] ||= [])
+
+          on_demand_resources_group = target.user_project.main_group["on_demand_resources"]
+          unless on_demand_resources_group
+            on_demand_resources_group = target.user_project.main_group.new_group("on_demand_resources", target.sandbox.root)
+          end
+
+          # Clean up previous on_demand_resources group
+          old_on_demand_resource_files = {}
+          on_demand_resources_group.groups.each do |group|
+            old_on_demand_resource_files[group.name] = []
+
+            group.files.each {|file|
+              old_on_demand_resource_files[group.name] << file
+            }
+          end
+          pb_target = target.user_project.objects_by_uuid[target.user_target_uuids[0]]
```

Comment:
`target.user_target_uuids[0]` is this means that `pb_target` always will be the first target? 

is that support mult targets？

just like
```
target 'TestLibrary_Tests' do
  pod 'TestLibrary', :path => '../'

  target 'TestLibrary' do  
end 
end

```